### PR TITLE
Run3-gex54 Use of DDCMS/DDutils.h in EcalCommonData/plugins/dd4hep

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Use of DDCMS/DDutils.h in EcalCommonData/plugins/dd4hep

#### PR validation:

Use test workflows in SimG4Core/Configuration/test

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/EcalCommonData/plugins/dd4hep/DDEcalEndcapAlgo.cc [hunk 1]
```diff
@@ -390,29 +396,30 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
...
+                                         << cms::convert2mm((crystal.centrePos().z() - ee.cryZOff)) << ")";
 #endif
           }
         }
       }
     }
   }
 
-  //** Loop over endcap columns
+  //** Loop over endcap columnsp
```
Comment (cvuosalo) [inline Geometry/EcalCommonData/plugins/dd4hep/DDEcalEndcapAlgo.cc:422]: Is "columnsp" a typo?

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**